internal/providers/terraform/azure/storage_table.go

```diff
@@ -25,19 +24,16 @@ func newStorageTable(d *schema.ResourceData) schema.CoreResource {
 	region := d.Region
 
 	accountReplicationType := "LRS"
-	accountKind := "StorageV2"
```

Comment:
We don't care about account kind for this resource since the pricing doesn't seem to be affected by it.